src/main.rs

```diff
@@ -1,10 +1,13 @@
 #![cfg_attr(feature = "strict", deny(warnings))]
 
+use std::process::exit;
 use clap::Parser;
 use cli::{CommandLine, Command};
 use commands::Execute;
 use log::{error, LevelFilter};
 
+use crate::api::HasExitCode;
```

Comment:
Is there a reason to use `crate` here?

```suggestion
use api::HasExitCode;
```

Reply:
Nope, vscode auto import